lua/orgmode/capture/init.lua

```diff
@@ -408,19 +408,17 @@ end
 ---@param arg_lead string
 ---@return string[]
 function Capture:autocomplete_refile(arg_lead)
-  local valid_filenames = self:_get_autocompletion_files(true)
+  local valid_files = self:_get_autocompletion_files(true)
 
-  if not arg_lead then
-    return vim.tbl_keys(valid_filenames)
+  if not arg_lead or #arg_lead == 0 then
+    return vim.tbl_keys(valid_files)
   end
-  local parts = vim.split(arg_lead, '/', { plain = true })
 
-  local selected_file = valid_filenames[parts[1] .. '/']
+  local filename = vim.split(arg_lead, '/', { plain = true })[1]
+  local selected_file = valid_files[filename .. '/']
 
   if not selected_file then
-    return vim.tbl_filter(function(file)
-      return file:match('^' .. vim.pesc(parts[1]))
-    end, vim.tbl_keys(valid_filenames))
+    return vim.fn.matchfuzzy(vim.tbl_keys(valid_files), vim.pesc(filename))
```

Comment:
We don't need `vim.pesc` here, it is used for lua pattern matching, but `matchfuzzy` is vimscript function.

Reply:
Thanks for the hint!
I removed it.

Btw. I hope you don't mind me force pushing.
Not sure what you'd consider best practice in this project.
But for now I didn't wanna bloat the commit history...

Reply:
> Btw. I hope you don't mind me force pushing.
> Not sure what you'd consider best practice in this project.
> But for now I didn't wanna bloat the commit history...

I force-push on PRs all the time and @kristijanhusak never complained. Because the PRs get squash-merged anyway I think it is no big deal. 🙂